bindings/rust/s2n-tls/src/raw/error.rs

```diff
@@ -210,6 +210,17 @@ unsafe fn cstr_to_str(v: *const c_char) -> &'static str {
     core::str::from_utf8_unchecked(bytes)
 }
 
+impl TryFrom<std::io::Error> for Error {
```

Comment:
is this being used?

Reply:
No. I initially wrote it for the shutdown tests, then realized I didn't need it. But customers definitely will need it, since all the io operations return io errors and they're silly hard to convert.